src/main/java/io/zold/api/Remote.java

```diff
@@ -37,4 +37,10 @@
      * @return The score
      */
     Score score();
+
+    /**
+     * Add a {@link Wallet} to this remote.
+     * @param wallet Wallet to be added to this remote
+     */
+    void add(Wallet wallet);
```

Comment:
@paulodamaso I think this should be named `push` as well. So that in the end, either the user "pushes a wallet to the network" or just "pushes a wallet to a remote"